test/e2e/testnets/testnet.go

```diff
@@ -28,7 +28,7 @@ type Testnet struct {
 
 func New(name string, seed int64, grafana *GrafanaInfo) (*Testnet, error) {
 	identifier := fmt.Sprintf("%s_%s", name, time.Now().Format("20060102_150405"))
-	if err := knuu.InitializeWithIdentifier(identifier); err != nil {
```

Comment:
This PR could have been split into two PRs, one for replacing namespace with scope and another for the re-ordering of the calls to the Commit and AddFolder.

Reply:
It was already split into two. I handled the deprecated usage in https://github.com/celestiaorg/celestia-app/pull/3347 which has already been merged. I'll rebase this on `main` to clarify it's not actually new in this PR.

Reply:
Done.